src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1092,6 +1119,31 @@ instance : Std.LawfulCommIdentity (α := BitVec n) (· ^^^ · ) (0#n) where
   ext i
   simp
 
+@[simp]
```

Comment:
In theory I suppose having xor injectivity could also be simplification rules, though Bitwuzla doesn't use it. I haven't thought too hard about why, but I suppose if they just transform xor into and+or there are other rules that can perform the same simplification.

Reply:
I'll add that in this change then.

Reply:
Why is what bitwuzla does different from what we could achieve with, say, simp priority levels that apply simplifications for individual operations first before applying multi-operation rewrites?